src/AdminModule/Components/AdminHome/AdminHome.tsx

```diff
@@ -1,7 +1,158 @@
-import React from 'react'
+import axios from 'axios';
+import React, { useEffect, useState } from 'react'
+import { baseUrl } from '../../../Constants/Components/Urls';
+import { PieChart } from '@mui/x-charts';
+import { Box, Container, Grid, Paper, Typography } from '@mui/material';
+import { WorkOutlineOutlined } from '@mui/icons-material';
+import { Link } from 'react-router-dom';
+
+
+
 
 export default function AdminHome() {
+
+  const [dashboardData,setDashboardData]=useState({});
+  // const data=[
+  //   {id:0,value:dashboardData?.bookings?.pending,label:'pending'},
+  //   {id:1,value:dashboardData?.bookings?.completed,label:'completed'},
+  // ]
+  async function getDashboardData() {
+    const token=localStorage.getItem('adminToken');
+    try {
+      const response= await axios.get(`${baseUrl}/admin/dashboard`,{headers:{Authorization:token}});
+      console.log(response?.data?.data);
+      setDashboardData(response?.data?.data);
+      
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  useEffect(()=>{
+    getDashboardData();
+  },[])
+
   return (
-    <div>AdminHome</div>
+    // <Box sx={{ flexGrow: 1 }}></Box>
+    <Grid container component="main" sx={{ height: "100vh" }}>
+      <Grid
+      container
+      columnGap={3}
+      direction="row"
+      justifyContent="center"
+      alignItems="center"
+      sx={{boxShadow:'none', 
+      // mx:3,pt:5
+    }}
+    >
+
+        <Grid item
+      xs={5}
+      sm={5}
+      md={3}
+      component={Paper}
+      elevation={6}
+      square
+      sx={{boxShadow:'none',bgcolor:'black',color:'white',borderRadius:'10px',textAlign:'center',alignContent:"center",py:2,height:'30%'}}
+      >
+       <Box sx={{display:'flex',justifyContent:'space-evenly',alignItems:'center'}}>
```

Comment:
We can make the whole box as a navigation. It is also unclear for users the an icon is clickable here. So making the whole box as a link for navigation and changing the cursor might help.

Also the black color  for each label doesn't match with colors in the page. You can look for similar designs for dashboard designs and follow one that match our design for a better look.
![image](https://github.com/SalmaAboArab/Hotel-Management-System/assets/64327685/9513afc5-3e7b-4e5c-9386-0fdfdaec243f)

Reply:
We can also decrease the space between between the labels and their pie charts. We also can add the values or percentage on top of the sectors like this Example:
https://mui.com/x/react-charts/pie/#labels